### ISSUE TITLE
i can use the slide only once ?

### ISSUE THREAD

**Issue by SalmanSoul** [dekatotoro/SlideMenuControllerSwift#229]
i can use it only at the beginning after that i can't slide ?

**Comment by sangadji** [comment]
Hi @SalmanSoul 
Did you mean you can only use it once and then after you change the view controller you cannot close the slidecontroller by slide anymore?  (only by tapping outside the slidecontroller). If its the case then I'm experiencing the same issue. Someone please help!

**Comment by SalmanSoul** [comment]
yup i think when we slide we need to call the same function that when you tap at the tap bar


i'm having tough week so if i get a free time i will try to do it


________________________________

**Comment by sangadji** [comment]
If you go back to the previous view controller would you still be able to slide the slidecontroller? If you are, I think I have found the solution
